app/assets/stylesheets/sufia/_fixedsticky.scss

```diff
@@ -0,0 +1,28 @@
+.fixedsticky {
+  position: -webkit-sticky;
+  position: -moz-sticky;
+  position: -ms-sticky;
+  position: -o-sticky;
+  position: sticky;
+}
+
+/* When position: sticky is supported but native behavior is ignored */
+.fixedsticky-withoutfixedfixed .fixedsticky-off,
+.fixed-supported .fixedsticky-off {
+  position: static;
+}
+
+.fixedsticky-withoutfixedfixed .fixedsticky-on,
+.fixed-supported .fixedsticky-on {
+  position: fixed;
+  left: $save-viz-widget-left;
```

Comment:
This seems like properties of the save widget -- maybe it should only style the save widget and not every fixedsticky?